_pages/library.md

```diff
@@ -827,3 +827,143 @@ function debug.traceback(msg: string?, level: number?): string
 ```
 
 Produces a stringified callstack of the given thread, or the current thread, starting with level `level`. If `msg` is specified, then the resulting callstack includes the string before the callstack output, separated with a newline. The format of the callstack is human-readable and subject to change.
+
+## buffer library
+
+Buffer is an object that represents a fixed-size mutable block of memory.
+
+All operations on a buffer are provided using the 'buffer' library functions.
+
+```
+function buffer.create(size: number): buffer
+```
+
+Creates a buffer of the requested size with all bytes initialized to 0.
+
+Size should be an integer number and cannot be negative.
```

Comment:
I would probably recommend noting offset/count/size restrictions once for all functions in the preamble above. Sizes should be non-negative integers, all buffer accesses outside of buffer bounds result in an error, etc.?

Reply:
Added.